module-apps/application-meditation/windows/MeditationOptionsWindows.cpp

```diff
@@ -122,9 +122,9 @@ void PreparationTimeWindow::onBeforeShow([[maybe_unused]] ShowMode mode, [[maybe
 unsigned int PreparationTimeWindow::getSelectedItemIndex() const
 {
     auto app                = static_cast<app::ApplicationMeditation *>(application);
-    const auto selectedTime = app->state->preparationTime;
-    for (unsigned int i = 0; i < preparationTimes.size(); ++i) {
-        if (selectedTime == preparationTimes[i]) {
+    const auto selectedTime = app->state->getPreparationTime();
+    for (unsigned int i = 0; i < Constants::Params::preparationTimes.size(); ++i) {
```

Comment:
This should probably be `size_t` - it would be safer so that in the future we can avoid a warning (and an error due to `-Werror`) if `size_t` would be e.g. `unsigned **long** int`

Reply:
done